bin/docker-image

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+build() {
+  ./bin/dist
+  docker build -t archway -m 4g .
+  docker save --output artifact-image.docker archway
+}
+
+case $1 in
```

Comment:
since there's only one functionality the function/switch statement can be removed (unless you foresee others?)